apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -163,6 +164,12 @@ userView { user } = React.fragment
   , classy DOM.div "col col-12 md-col-6 lg-col-6" [ subscriptionsView ]
   ]
   where
+    subsSorted = foldl canceledLast [] user.subs
```

Comment:
I wonder if we should start adding more docstrings, feels like some context would help coming back to this after some time